validation/validation_test.go

```diff
@@ -0,0 +1,67 @@
+package validation_test
+
+import (
+	"testing"
+
+	"github.com/preslavmihaylov/todocheck/validation"
+	"github.com/preslavmihaylov/todocheck/config"
+)
+
+func TestInvalidOrigins(t *testing.T) {
+	invalidConfigPaths := []string{
+		"../testing/test_configs/invalid_github_https.yaml",
```

Comment:
Noticed that I changed the path to avoid moving the .yaml files. Maybe they should be moved?